app/models/permissible_value.rb

```diff
@@ -10,8 +10,13 @@ def self.get_value_list(category)
     PermissibleValue.where(category: category).pluck(:value)
   end
 
+  # Get a hash of PermissibleValue keys as they keys and values as values
+  def self.get_hash(category)
```

Comment:
`#get_hash` uses the standard format { key => value } but `#get_value_key_hash` uses a { value => key } format